dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_5_MA2.rst

```diff
@@ -23,17 +23,54 @@ Setup:
 - On the **Home** tab set the instrument to **PSI**
 - The load current run button should be greyed out
 - Load **dolly 1529** using the **Browse** button
+- Set `Rebin` to `Fixed` and enter a value of `5`
 - You will get 4 sets of data that curve upwards
 - In the **Grouping** tab there will be 4 groups defined
 
 
-Test 1:
--------
+Auto Background Corrections:
+----------------------------
+
+- Go to the **Corrections** tab
+- For the ``Background`` select ``Auto``
+- The plots will now average around 0
+- The values should be (approximatley):
+	- Forw: ``19.2``
+	- Back: ``93.6``
+	- Left: ``104.4``
+	- Rite: ``77.2``
```

Comment:
```suggestion
	- Right: ``77.2``
```

Reply:
That is what the group is called, they are only able to write 4 letters.